### PR TITLE
Update issue templates

### LINKED ISSUE #21: Community standards: Add Issue templates
That facilitates enormously the management of issues and guides anyone opening an issue into providing the required information. Which templates to add is up to you, but you could have a look at those used by the PyBaMM project for inspiration:

[PyBaMM/.github/ISSUE_TEMPLATE at develop · pybamm-team/PyBaMM](https://github.com/pybamm-team/PyBaMM/tree/develop/.github/ISSUE_TEMPLATE)

More on Community Standards: [Community Standards (github.com)](https://github.com/ImperialCollegeLondon/virtual_rainforest/community)

### PR DESCRIPTION
**Pull request by vgro**
First version of issue templates, default templates
Fixes #21

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (dalonsoa) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: If it's a bug, probably we want a label that says "bug", right?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (dalonsoa) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Is this section relevant for this code?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (dalonsoa) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: No need browser information

**Review by dalonsoa — CHANGES_REQUESTED** [review]
The bug template probably needs some refinement.

**Review by TaranRallings — APPROVED** [review]
Looks good to me!

**Review by dalonsoa — APPROVED** [review]
All good!

**Pull request merged**